packages/mobile/src/transactions/TransactionFeedItem.tsx

```diff
@@ -0,0 +1,99 @@
+import Touchable from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors.v2'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { MoneyAmount, TokenTransactionType } from 'src/apollo/types'
+import CurrencyDisplay, { FormatType } from 'src/components/CurrencyDisplay'
+import { Namespaces } from 'src/i18n'
+import { TransactionStatus } from 'src/transactions/reducer'
+
+interface Props {
+  type: TokenTransactionType
+  amount: MoneyAmount
+  title: string
+  info: React.ReactNode | string | null | undefined
+  icon: React.ReactNode
+  timestamp: number
+  status: TransactionStatus
+  onPress: () => void
+}
+
+export function TransactionFeedItem(props: Props) {
+  const { t } = useTranslation(Namespaces.walletFlow5)
+
+  const { type, amount, title, info, icon, status, onPress } = props
+
+  const isReceived = new BigNumber(amount.value).isPositive()
+  const isPending = status === TransactionStatus.Pending
+
+  let subtitle = info
+  if (isPending) {
+    subtitle = t('confirmingTransaction')
+  }
```

Comment:
I think this would be a bit more readable
```suggestion
  const subtitle = isPending ? t('confirmingTransaction') : info
```